documentation/autoinstall-reference.md

```diff
@@ -416,6 +461,58 @@ The installer will update the target with debconf set-selection values. Users wi
 
 A list of packages to install into the target system. More precisely, a list of strings to pass to "`apt-get install`", so this includes things like task selection (`dns-server^`) and installing particular versions of a package (`my-package=1-1`).
 
+### kernel
+
+**type:** mapping (mutually exclusive), see below
+**default:** default kernel
+**can be interactive:** no
+
+Which kernel gets installed. Either the name of the package or the name of the flavor must be specified.
+
+#### package
+
+**type:** string
+
+The name of the package, e.g., `linux-image-5.13.0-40-generic`
+
+#### flavor
+
+**type:** string
+
+The flavor of the kernel, e.g., `generic` or `hwe`.
+
+### timezone
+
+**type:** string
+**default:** no timezone
+**can be interactive:** no <!-- TODO Should it be yes? There is no screen in the UI but a HTTP API -->
```

Comment:
I think "no" is the correct answer here.  No screen (yet).  There are aspirations to add something like that - the notion of having a time setting in the top orange bar has been thrown around.